TECH-LEADS-CHECKLIST.md

```diff
@@ -1,71 +1,85 @@
+
 - [ ] The master branch is locked.
 - [ ] Merges are done through PRs.
-- [ ] PRs reference related work items. 
+- [ ] PRs reference related work items.
 - [ ] Commit history is consistent and commit messages are informative (what, why).
 - [ ] Secrets are not part of the commit history or made public. (see [Credential scanning](./continuous-integration/credential-scanning/README.md))
 - [ ] Public repositories follow the [OSS guidelines](source-control/git.md), see `Required files for public repositories`.
 
 More details on [Source Control](source-control/readme.md)
 
 ## Work Item Tracking
+
 - [ ] All items are tracked in AzDevOps (or similar).
 - [ ] The board is organized (swim lanes, feature tags, technology tags).
 
 ## Testing
-- [ ] Unit tests cover the majority of all components (>90% if possible). 
-- [ ] Integration tests run to test the solution e2e. 
+
+- [ ] Unit tests cover the majority of all components (>90% if possible).
+- [ ] Integration tests run to test the solution e2e.
 
 More details on [Unit Testing](test-first-development/unit-testing/readme.md)
 
 ## CI/CD
-- [ ] Project runs CI with automated build and test on each PR. 
+
+- [ ] Project runs CI with automated build and test on each PR.
 - [ ] Project uses CD to manage deployments to a replica environment before PRs are merged to master.
-- [ ] Master is always shippable. 
+- [ ] Master is always shippable.
+
+## Security - TO DO
 
-## Security - TO DO! 
-- [ ] Access control. 
-- [ ] Separation of concerns. 
+- [ ] Access control.
+- [ ] Separation of concerns.
 - [ ] Robust treatment of secrets.
 - [ ] Encryption for data in transit (and if necessary at rest) and password hashing.
 
 ## Observability
-- [ ] Application faults and errors are logged. 
+
+- [ ] Application faults and errors are logged.
 - [ ] Metrics collected to identify health of the services.
 - [ ] Latency for servicing a request is logged, as appropriate for the solution.
-- [ ] Significant business and functional events that are important are tracked and related metrics collected. 
+- [ ] Significant business and functional events that are important are tracked and related metrics collected.
 - [ ] Logging configuration can be modified without code changes (eg: debug mode).
-- [ ] GDPR compliance is ensured regarding PII (personal identifiable information). 
+- [ ] GDPR compliance is ensured regarding PII (personal identifiable information).
 
 ## Agile/Scrum
+
 - [ ] Scrum Master (fixed/rotating) to run standup daily.
 - [ ] Agile process clearly defined within team.
-- [ ] Tech Lead (+ PO/Others) have responsability for backlog management and grooming.
+- [ ] Tech Lead (+ PO/Others) have responsibility for backlog management and grooming.
 - [ ] Working agreement between members of team and customer.
 
 ## Design Reviews
+
 - [ ] Process for conducting design reviews.
+- [ ] Including design review process in the [Working Agreement](/team-agreements/working-agreements/readme.md)
```

Comment:
Would perhaps be better to include in the first bullet (above) eg: "Process for conducting design reviews is documented in [Working Agreement](/team-agreements/working-agreements/readme.md)

Reply:
Updated.